JustSaying.UnitTests/JustSaying.UnitTests.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I was going to attempt to unify the packages using a `nuspec` file, but unifying the projects would make things much easier 😃